config/api.yml

```diff
@@ -2315,6 +2316,53 @@
       :post:
       - :name: remove
         :identifier: svc_catalog_provision
+  :cloud_templates:
+    :description: Cloud Templates
+    :identifier: template
+    :options:
+      - :collection
+      - :subcollection
+    :verbs: *gpd
+    :klass: ManageIQ::Providers::CloudManager::Template
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: miq_template_show_list
+      :post:
+      - :name: query
+        :identifier: miq_template_show_list
```

Comment:
Is there a reason why you are using `miq_template_show_list` instead of the `image_show_list`? The product features seem to be inconsistent

Reply:
@jntullo yes, replaced it